src/mono/browser/build/BrowserWasmApp.targets

```diff
@@ -532,6 +532,7 @@
       <Output TaskParameter="ExitCode" PropertyName="_EmccLinkStepExitCode" />
     </Exec>
     <ItemGroup>
+      <!-- TODO add only if changed -->
```

Comment:
If the file part of current build, although unchanged/untouched, it should be included in FileWrites

Reply:
why ?